src/setup-beam.js

```diff
@@ -17,21 +18,49 @@ async function main() {
   const elixirSpec = core.getInput('elixir-version', { required: false })
   const gleamSpec = core.getInput('gleam-version', { required: false })
   const rebar3Spec = core.getInput('rebar3-version', { required: false })
+  const shouldMixRebar = core.getInput('install-rebar', { required: false })
+  const shouldMixHex = core.getInput('install-hex', { required: false })
+  process.env.HEX_MIRROR = core.getInput('hexpm-mirror', { required: false })
```

Comment:
Can we pass this as a parameter to the installer function and subsequently to the installer scripts, as is done for other stuff? Reading from the env. to have to search for what is setting a given variable is confusing and error prone.

Reply:
FYI the environment variable is not only used by the installer scripts, it is also used by the mix tool invoked directly from js.

Reply:
I see. Can we then not do a `HEX_MIRROR=... mix` from that call, in this case? I'd really prefer to not have to use env. variables to pass values around.

Reply:
> @pguyot, I think this is getting close to ready.
> 
> We're just missing some small things:
> 
> * [ ]  [comment](https://github.com/erlef/setup-beam/pull/156#discussion_r1026960101)
This was fixed  in 634fc94ae5b9d0c83f7ff2928dfbec41ddddc9ed.

> * [ ]  [comment](https://github.com/erlef/setup-beam/pull/156#discussion_r1028523824)

Sorry, this PR is not about retries. We shouldn't retry mirrors.
In fact, I even removed the caching feature as it doesn't make sense for mirrors as well.

> * [ ]  [update suggestion](https://github.com/erlef/setup-beam/pull/156#discussion_r1028515853)
Commit created.

> * [ ]  [update suggestion](https://github.com/erlef/setup-beam/pull/156#discussion_r1028530097)
Because cache feature has been removed, it is no longer necessary to modify the semantics of this function and therefore I removed the documentation I wrote.

> * [ ]  [update suggestion](https://github.com/erlef/setup-beam/pull/156#discussion_r1028530246)
Likewise.

> * [ ]  [update suggestion](https://github.com/erlef/setup-beam/pull/156#discussion_r1028532269)
Outdated.

> * [ ]  [comment](https://github.com/erlef/setup-beam/pull/156#discussion_r1028533044)
This is how options work with actions. The API provides a multiline parsing. I believe lists are not supported.

> * [ ]  finally, there's now merge conflicts
My approach is to rebase and force push, but this is not what you asked for. Please advise.

Reply:
Will be resolved via a comment.